cross_compiler.py

```diff
@@ -2025,7 +2056,7 @@ def cchdir(self,dir):
 			'PKGCONFIG' : 'pkg-config',
 		},
 		'depends_on' : [
-			'libffmpeg',
+			'libffmpeg', # 'liblsw', # ... lsw for x264 requires --enable-avresample which is deprecated # ... gave up on 'libgpac'
```

Comment:
I used to add comments like this as well, but it's kinda ugly, so I stopped, remove those please and its alright. (same goes to the  2018.06.10 *** ones)